### DISCUSSION TITLE
Python GIS libraries possible?

### DISCUSSION THREAD

**Discussion by Geogch** [modular/modular#157] [Q&A] [answered]
Hi. 
Would it be possible to incorporate the Python GIS libraries?
Such as ArcPy or geopandas?

Thanks

**Comment by lattner** [comment]
Mojo can already talk to arbitrary python libraries, do you mean in the notebook?

**Reply by Geogch** [reply]
Yes, I mean in the notebook. 
How do we run mojo on desktop/in a terminal? (apologies for the novice question)

**Comment by lattner** [comment, marked as answer]
We're working on enabling local mojo, but it isn't ready quite yet

**Reply by Geogch** [reply]
Ok great! Hopefully it's complete soon.
thanks!
